guides/05_chatbots/08_creating-a-website-widget-from-a-gradio-chatbot.md

```diff
@@ -0,0 +1,210 @@
+# 🚀 Creating a Website Chat Widget with Gradio 🚀
+
+Tags: CHAT, DEPLOY, WEB
+
+You can make your Gradio Chatbot available as an embedded chat widget on your website, similar to popular customer service widgets like Intercom. This is particularly useful for:
+
+- Adding AI assistance to your documentation pages
+- Providing interactive help on your portfolio or product website
+- Creating a custom chatbot interface for your Gradio app
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/gradio-guides/Screen%20Recording%202024-12-19%20at%203.32.46%E2%80%AFPM.gif)
+
+## How does it work?
+
+The chat widget appears as a small button in the corner of your website. When clicked, it opens a chat interface that communicates with your Gradio app via the JavaScript Client API. Users can ask questions and receive responses directly within the widget.
+
+![Widget Demo GIF placeholder]
+
+## Prerequisites
+
+* A running Gradio app (local or on Hugging Face Spaces). In this example, we'll use the [Gradio Studio Space](https://huggingface.co/spaces/abidlabs/gradio-playground-bot), which helps generate code for Gradio apps based on natural language descriptions.
```

Comment:
why is it called Gradio Studio Space in this guide? 
```suggestion
* A running Gradio app (local or on Hugging Face Spaces). In this example, we'll use the [Gradio Playground Space](https://huggingface.co/spaces/abidlabs/gradio-playground-bot), which helps generate code for Gradio apps based on natural language descriptions.
```

Reply:
whoops